### PR TITLE
Add merro60 pcb firmware

### PR DESCRIPTION
**Pull request by gaclee3b**
## Description

Adding firmware for a new custom 60% PCB. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by gaclee3b** [comment]
I figured out how to preview the info.json - either I was using the wrong key combo to get the previewer or it was because I switched from safari to chrome, but I got it to load and reviewed the layouts and they look good to me.

File: keyboards/chlx/merro60/config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4358 // "CX" - chlx
+#define PRODUCT_ID      0x0601
+#define DEVICE_VER      0x0100
+#define MANUFACTURER    chlx
+#define PRODUCT         chlx merro60
+#define DESCRIPTION     tofukorn conversion pcb
```
Comment (drashna) [inline keyboards/chlx/merro60/config.h:26]:
```suggestion
```

File: keyboards/chlx/merro60/keymaps/ansi/keymap.c [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/ansi/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/keymaps/hhkb/keymap.c [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/hhkb/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/keymaps/iso/keymap.c [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/iso/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/keymaps/tsangan/keymap.c [hunk 6]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/tsangan/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/merro60.c [hunk 8]
```diff
@@ -0,0 +1,2 @@
+#include "merro60.h"
```
Comment (drashna) [inline keyboards/chlx/merro60/merro60.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/merro60.h [hunk 9]
```diff
@@ -0,0 +1,172 @@
+#pragma once
```
Comment (drashna) [inline keyboards/chlx/merro60/merro60.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/chlx/merro60/rules.mk [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE  = no      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE       = yes     # USB Nkey Rollover
+BACKLIGHT_ENABLE  = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE   = no      # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE  = no      # Enable Bluetooth
+AUDIO_ENABLE      = no      # Audio output
+
+LAYOUTS = ansi default hhkb iso tsangan via
```
Comment (drashna) [inline keyboards/chlx/merro60/rules.mk:24]: This is for community layouts, specifically. 

If they don't match what is in /layouts/default/, then they shouldn't be set here. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by gaclee3b — COMMENTED** [review]
implement suggested removals

File: keyboards/chlx/merro60/keymaps/via/merro60.json [hunk 11]
```diff
@@ -0,0 +1,229 @@
+{
```
Comment (drashna) [inline keyboards/chlx/merro60/keymaps/via/merro60.json:1]: This file doesn't belong in the QMK repo, and will not be used by QMK.   It needs to be removed, and PR'ed to `the-via`'s keyboards repo.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/chlx/merro60/keymaps/via/merro60.json (hunk 11 above)
Comment (gaclee3b) [reply keyboards/chlx/merro60/keymaps/via/merro60.json]: removed file

Comment (gaclee3b) [reply keyboards/chlx/merro60/keymaps/via/merro60.json]: PR'd JSON file to via

File: keyboards/chlx/merro60/readme.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+
+merro60.1, merro60.1.1 PCB
+
+A standard 60% format keyboard pcb with USB-C and unified daughterboard compatibility.
+Has modified cutouts to allow usage of certain tray mount cases into gummy gasket o-ring mount.
+
+Keyboard Maintainer: [Alexander Lee](https://github.com/gaclee3b)
+Hardware Supported: chlx PCB merro60.1, merro60.1.1
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/chlx/merro60/readme.md:11]: Could you add instructions on how to jump into the bootloader?

**Review by drashna — APPROVED** [review]

File: keyboards/chlx/merro60/keymaps/ansi/keymap.c [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______),
+
+[3] = LAYOUT_ansi(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______)
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/keymaps/ansi/keymap.c:49]:
```suggestion
```

File: keyboards/chlx/merro60/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______),
+
+[3] = LAYOUT_default(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______)
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/keymaps/default/keymap.c:49]:
```suggestion
```

File: keyboards/chlx/merro60/keymaps/hhkb/keymap.c [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+           _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+           _______, _______,                   _______,                                              _______, _______),
+
+[3] = LAYOUT_hhkb(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+           _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+           _______, _______,                   _______,                                              _______, _______)
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/keymaps/hhkb/keymap.c:49]:
```suggestion
```

File: keyboards/chlx/merro60/keymaps/iso/keymap.c [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______),
+
+[3] = LAYOUT_iso(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                     _______, _______, _______, _______)
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/keymaps/iso/keymap.c:49]:
```suggestion
```

File: keyboards/chlx/merro60/keymaps/tsangan/keymap.c [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+           _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                              _______, _______, _______),
+
+[3] = LAYOUT_tsangan(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+           _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______,                   _______,                                              _______, _______, _______)
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/keymaps/tsangan/keymap.c:49]:
```suggestion
```

File: keyboards/chlx/merro60/readme.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+
+Enter bootloader mode by either option -
+- When unplugged, holding the top-left key (typically ESC) while plugging in the keyboard
+- When plugged, pressing and releasing the reset button on the pcb bottom behind the CAPSLOCK / A-key region
+
+Make example for this keyboard (after setting up your build environment):
+
+    make chlx/merro60:default
+    make chlx/merro60:via
+    make chlx/merro60:all
```
Comment (fauxpark) [inline keyboards/chlx/merro60/readme.md:19]:
```suggestion
```

File: keyboards/chlx/merro60/rules.mk [hunk 19]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE    = no      # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE  = no      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE       = yes     # USB Nkey Rollover
+BACKLIGHT_ENABLE  = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE   = no      # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE  = no      # Enable Bluetooth
+AUDIO_ENABLE      = no      # Audio output
+
```
Comment (fauxpark) [inline keyboards/chlx/merro60/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**